test/functional/test_runner.py

```diff
@@ -544,6 +544,11 @@ def run_tests(*, test_list, src_dir, build_dir, tmpdir, jobs=1, enable_coverage=
                 logging.debug("Early exiting after test failure")
                 break
 
+            shutil_disk_usage_stat_ = shutil.disk_usage(testdir)
+            if shutil_disk_usage_stat_.free < 1:
```

Comment:
I think this can be bumped to 16 M, because it is not possible to create a datadir that is smaller than that.

The first block file will pre-allocate 16M:

```
16M	/tmp/aaa/regtest/blocks/blk00000.dat